.golangci-lint.yml

```diff
@@ -105,6 +105,7 @@ issues:
         - gocyclo
         - errcheck
         - gosec
+        - wrapcheck
```

Comment:
It seems that `//nolint:wrapcheck` doesn't needed anymore

Reply:
it is only disabling wrapcheck on _test files. And I'm pretty sure that we don't have `//nolint:wrapcheck` in test files